package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "my-crypto-metrics",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.6.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.5.0",
+    "chart.js": "^3.0.0",
+    "font-awesome": "^4.7.0",
+    "react": "^16.14.0",
+    "react-chartjs-2": "^3.0.0",
+    "react-dom": "^16.14.0",
+    "react-redux": "^7.2.6",
+    "react-router-dom": "^6.0.0",
+    "react-scripts": "^4.0.3",
+    "redux-thunk": "^2.4.2",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
### Really? Because I don't seem to have any issue with running 'npm start' on my end...

<div align="center">

![](https://github.com/CVILLA09/my-crypto-metrics/blob/develop/assets/start-test.gif?raw=true)

</div>